app/assets/stylesheets/views/_covid.scss

```diff
@@ -292,6 +292,7 @@
 
 .covid__page-guidance-link--white {
   color: govuk-colour("white");
+  padding-left: 30px;
```

Comment:
Could this be `govuk-spacing(6)` instead?

Reply:
😮